src/mumble/TalkingUI.h

```diff
@@ -10,6 +10,7 @@
 #include <QtCore/QHash>
 #include <QtCore/QSet>
 #include <QtGui/QIcon>
+#include <QtGui/QColor>
```

Comment:
because QColor's name() and isValid() functions are called

Reply:
The correct include should actually be `#include <QColor>`. The other include directives are wrong in that regard as well - no need to copy that :D